frappe/desk/form/assign_to.py

```diff
@@ -66,10 +66,7 @@ def add(args=None):
 	notify_assignment(d.assigned_by, d.owner, d.reference_type, d.reference_name, action='ASSIGN',\
 			 description=args.get("description"), notify=args.get('notify'))
 
-	if not args.get("bulk_assign"):
```

Comment:
`bulk_assign` is handled by `add_multiple` so this is not required here. I've tested on my local, it works with these changes.